management_api_app/db/repositories/workspace_templates.py

```diff
@@ -40,3 +42,20 @@ def get_workspace_template_names(self) -> List[str]:
         print(workspace_templates)
         workspace_template_names = [template["name"] for template in workspace_templates]
         return list(set(workspace_template_names))
+
+    def create_workspace_template_item(self, workspace_template_create: WorkspaceTemplateInCreate):
+        item_id = str(uuid.uuid4())
+        resource_template = ResourceTemplate(
+            id=item_id,
+            name=workspace_template_create.name,
+            description=workspace_template_create.description,
+            version=workspace_template_create.version,
+            properties=workspace_template_create.parameters,
+            resourceType=workspace_template_create.resourceType,
```

Comment:
The resource type for workspace templates should always be workspace so this does not need to be in the input.

Reply:
fixing